getting-started/getting-started.md

```diff
@@ -46,11 +60,19 @@ In the pom.xml or build.gradle, add this entry to the `plugins` section to apply
 {% endtab %}
 
 {% tab title="Gradle" %}
+
+* Add the rewrite plugin to the `plugins` section of your `build.gradle` file
+* Make sure `mavenCentral()` is included in the `repositories` section
+* Add a `rewrite` section that will be filled in later
+
+Your file should look similar to:
+
 {% code title="build.gradle" %}
 ```groovy
 plugins {
-    id("java")
-    id("org.openrewrite.rewrite") version("5.32.0")
+    id 'java'
+    id 'maven-publish'
+    id 'org.openrewrite.rewrite' version '5.32.0'
```

Comment:
The changes you've made to omit parenthesis and prefer single-quoted to double-quoted strings _do_ make these more idiomatic groovy. But Gradle scripts can be written in Groovy or Kotlin. And Kotlin has no single-quoted string and does not allow parenthesis to be omitted in these circumstances.

But part of why I used double-quoted strings and included parenthesis is that the resulting code would, generally, work _either_ in groovy or kotlin scripts. If we're going to make the groovy in these code snippets more idiomatic, which is obviously not a bad thing, we might consider _also_ including gradle kotlin script examples.

Reply:
That's fair re the Kotlin side of things. The reason I switched this, though, is because when I checked out the sample repository that we suggest, it doesn't use the double quotes and parenthesis. I figure if someone was following along, it would be nicer to have the file look like it would in the sample repository. 

Would you mind explaining what you mean by `also including gradle kotlin script examples`? I'm not sure I follow that.

Reply:
Going to merge this for now but happy to update later once I better understand the ask :)